### ISSUE TITLE
[Bug]: The media player prevents screen scrolling

### ISSUE THREAD

**Issue by ghost** [FreeTubeApp/FreeTube#2778]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for open and closed issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.

### Describe the bug

Trying to screen scroll with the touchpad or mouse wheel does not work if the pointer is over the media player. Of course, with the setting to control the volume with scrolling disabled.

### Expected Behavior

Screen scrolling.

### Issue Labels

accessibility issue, usability issue

### FreeTube Version

v0.17.1

### Operating System Version

Linux Mint 21 Cinnamon

### Installation Method

Flathub

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Could u try out [the latest nightly build](https://docs.freetubeapp.io/development/nightly-builds)? 0.17.1 is running on electron v16 and nightly build is already on v21/and has better support for wayland users.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Couldnt reproduce this on v0.18.0. Both flatpak and rpm worked on fedora. Can u check if v0.18.0 resolved this for u?
